pkg/registry/networking/networkpolicy/strategy.go

```diff
@@ -59,6 +66,11 @@ func (networkPolicyStrategy) PrepareForUpdate(ctx context.Context, obj, old runt
 	newNetworkPolicy := obj.(*networking.NetworkPolicy)
 	oldNetworkPolicy := old.(*networking.NetworkPolicy)
 
+	// Update is not allowed to set status when the operation is not directed to status subresource
```

Comment:
Should we feature gate this even on the "Spec" method (not status)?

Reply:
yes